dotCMS/src/integration-test/java/com/dotcms/enterprise/rules/RulesAPIImplIntegrationTest.java

```diff
@@ -82,12 +79,80 @@ public void shouldSaveCondition() throws DotSecurityException, DotDataException
         rulesAPI.saveConditionGroup(conditionGroup, user, false);
 
         List<Rule> allRules = rulesAPI.getAllRules(user, false);
+        checkCondition(user, conditionGroup, allRules);
+    }
+
+    /**
+     * Method to Test: {@link RulesAPIImpl#saveConditionGroup(ConditionGroup, User, boolean)}
+     *                  and {@link RulesAPIImpl#saveCondition(Condition, User, boolean)}
+     * When:Try to create a Group Condition and a Condition in a page's rule with PuBLISH permission over the page
+     * Should: save the rule
+     *
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldSaveConditionInRulesPage() throws DotSecurityException, DotDataException {
+        final Role role = new RoleDataGen().nextPersisted();
+        final User user = new UserDataGen().roles(role).nextPersisted();
+        final Host host = new SiteDataGen().nextPersisted();
+
+        final Template template = new TemplateDataGen().nextPersisted();
+        final HTMLPageAsset htmlPageAsset = new HTMLPageDataGen(host, template)
+                .nextPersisted();
+
+        addPermission(role, htmlPageAsset, PermissionAPI.PERMISSION_PUBLISH);
+        addRulesPermission(role, host, true);
+        final Rule rule = new RuleDataGen().page(htmlPageAsset).nextPersisted();
+
+        final ConditionGroup conditionGroup = new ConditionGroupDataGen().rule(rule).next();
+        this.addRulesPermission(role, host, true);
+
+        //Saving and testing GroupCondition
+        rulesAPI.saveConditionGroup(conditionGroup, user, false);
+
+        List<Rule> allRules = rulesAPI.getAllRules(user, false);
+        checkCondition(user, conditionGroup, allRules);
+    }
+
+    /**
+     * Method to Test: {@link RulesAPIImpl#saveConditionGroup(ConditionGroup, User, boolean)}
+     *                  and {@link RulesAPIImpl#saveCondition(Condition, User, boolean)}
+     * When:Try to create a Group Condition and a Condition in a page's rule without PuBLISH permission over the page
+     * Should: Throw a {@link DotSecurityException}
+     *
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test(expected = DotSecurityException.class)
+    public void shouldSaveConditionInRulesPageShouldNotWork() throws DotSecurityException, DotDataException {
+        final Role role = new RoleDataGen().nextPersisted();
+        final User user = new UserDataGen().roles(role).nextPersisted();
+        final Host host = new SiteDataGen().nextPersisted();
+
+        final Template template = new TemplateDataGen().nextPersisted();
+        final Folder folder = new FolderDataGen().site(host).nextPersisted();
+        final HTMLPageAsset htmlPageAsset = new HTMLPageDataGen(host, template)
+                .folder(folder)
+                .nextPersisted();
+
+        addPermission(role, htmlPageAsset, PermissionAPI.PERMISSION_PUBLISH);        addRulesPermission(role, host, true);
```

Comment:
Looks like this line shouldn't be there if you don't want publish perms on the page

Reply:
done https://github.com/dotCMS/core/pull/18201/commits/a3d5384f514af695b89fdbc549338def9a8cce2c#diff-9baded4fb0fb63e2648d9c9c4380ec1aL145